src/components/Navbar.jsx

```diff
@@ -0,0 +1,22 @@
+import { Navbar, Nav, Container } from 'react-bootstrap';
+import { Link } from 'react-router-dom';
+
+
+export default function Navigationbar(){
+    return (
+    <>
+    <Navbar bg="primary" variant="dark" expand="lg">
+    <Container>
+    <Navbar.Brand as={Link} to={'/'}>Playing For Keeps</Navbar.Brand>
+    <Navbar.Toggle aria-controls="basic-navbar-nav" />
+    <Navbar.Collapse id="basic-navbar-nav">
+    <Nav className="ms-auto">
+    <Nav.Link as={Link} to={'/'}>Home</Nav.Link>
+    <Nav.Link as={Link} to={'/login'}>Login</Nav.Link>
+    </Nav>
+    </Navbar.Collapse>
+    </Container>
```

Comment:
formatting is off

Reply:
Try adding this to vscode to do it automatically https://levelup.gitconnected.com/automatically-format-code-in-visual-studio-code-when-working-with-react-c48674a12dc5

Reply:
Should go:

when there is a <> everything inside is indented 2 spaces until the closing tag of </>

so 
`
<contatiner>
  <navbar>
  </navbar>
</container>
`